proxy/api/protocol.go

```diff
@@ -28,6 +29,18 @@ type header struct {
 	flags  uint32
 }
 
+const maxPayloadLength = 1 * 1024 * 1024
```

Comment:
I was lazy yesterday, but it seems hyperstart will realloc the receive buffer (without any check one the new size, ooops) https://github.com/clearcontainers/hyperstart/blob/0.7.0-clearcontainers/src/init.c#L1193